docs/layout.md

```diff
@@ -1,49 +1,40 @@
-Navigating Remix
-================
-Remix IDE is comprised of three panels and a terminal.
+# Navigating Remix
 
-![](images/a-layout1c.png)
+Remix IDE consists of four core components: the icon panel, side panel, main panel, and terminal. Since Remix v0.69.0, it also includes a top bar.
```

Comment:
I think it should be:
Remix's layout consists of the following components: top bar,  icon panel, side panel, main panel, and terminal.